MantidQt/MantidWidgets/src/DataProcessorUI/GenericDataProcessorPresenter.cpp

```diff
@@ -1409,20 +1442,18 @@ void GenericDataProcessorPresenter::plotGroup() {
     throw std::runtime_error("Can't plot group.");
 
   // Set of workspaces to plot
-  std::set<std::string> workspaces;
+  std::set<QString> workspaces;
   // Set of workspaces not found in the ADS
-  std::set<std::string> notFound;
+  QSet<QString> notFound;
 
-  const auto items = m_manager->selectedData();
+  auto const items = m_manager->selectedData();
```

Comment:
```
$ grep -rn 'auto const' | wc -l
98
$ grep -rn 'const auto' | wc -l
4605
$ grep -rn 'const int' | wc -l
2654
$ grep -rn 'int const' | wc -l
39
```


Running grep on the codebase reveals that although const-on-the-right is used, it is used two orders of magnitude less frequently than const-on-the-left. If we were to adopt an on-the-right style it is likely that we would need tool support of some kind (in which case clang's libTooling might be an option).